_posts/2025-06-05-resultado-motor-decisiones.md

```
@@ -0,0 +1,40 @@
+
+Podrás también filtrar los resultados en:
+
+- Todos
+- Aprobado
+- Rechazado
+- Revisión
+- Error
+
+Esto te permitirá ver cada uno de los resultados de manera clasificada.
+
+#### Aquí te mostramos como acceder.
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)